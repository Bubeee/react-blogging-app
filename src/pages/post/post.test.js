import React from 'react';
import { PostForm } from '.';

describe('PostForm', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<PostForm />);

    expect(wrapper).toMatchSnapshot();
  });
});
