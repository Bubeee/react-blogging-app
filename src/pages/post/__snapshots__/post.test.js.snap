// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostForm should render correctly 1`] = `
<div>
  <Header />
  <div
    className="post-form__container"
  >
    <form
      onSubmit={[Function]}
    >
      <label
        className="post-form__label"
        htmlFor="title"
      >
        Title
      </label>
      <input
        className="post-form__input"
        name="title"
        placeholder="Enter the title"
        type="text"
      />
      <label
        className="post-form__label"
        htmlFor="content"
      >
        Content
      </label>
      <textarea
        className="post-form__textarea"
        name="content"
        placeholder="Enter the content"
      />
      <input
        className="post-form__submit"
        type="submit"
        value="Add"
      />
    </form>
  </div>
  <Footer />
</div>
`;
